content/_code-samples/key-derivation/key-derivation.py

```diff
@@ -0,0 +1,327 @@
+#!/usr/bin/env python3
```

Comment:
Putting a hyphen in the file name makes it more difficult to import (I had to look up how do it with `importlib`)